spec/lib/lightning/api_spec.rb

```diff
@@ -1,186 +1,184 @@
-require 'rails_helper'
+# frozen_string_literal: true
 
-describe Lightning::Api do
+require 'rails_helper'
 
-  dummy_user = User.create(name: 'Dummy User')
+RSpec.describe Lightning::Api do
+  let(:user) { User.create(name: 'Dummy User') }
 
-  context '.create' do
+  describe '.create' do
     it 'creates the feature' do
-      Lightning::Api.create!('create_feature', 'New feature for testing')
-      expect(Lightning::Api.get('create_feature')).to be_an_instance_of Lightning::Feature
+      described_class.create!('create_feature', 'New feature for testing')
```

Comment:
I'm curious where this `described_class` is defined

Reply:
It is an rspec construct: https://relishapp.com/rspec/rspec-core/docs/metadata/described-class
Basically, it refers to the class described in the first line, in the future if we decide to change the definition of the class, we won't have to replace every instance.